frontend/src/App.js

```diff
@@ -36,6 +36,17 @@ function App(props){
       ) 
    }
 
+   function getProfessorsMatching(req){
+      const prof_matching = [];
+      const response = axios.get("http://localhost:5001/api/professor", { params: { name: req.body } })
+      // for over all the indices of the response table, and get each name and add it to the prof_matching[]
+      for(let i = 0; i < response.length; i++){
+         prof_matching.push(response[i].name);
+      }
+      // const names = response.data.name
+
```

Comment:
Does this function return the professors that match? It's supposed to, I belive.

Reply:
Yes it does, just finished testing!